src/consent-manager-builder/index.tsx

```diff
@@ -230,14 +228,16 @@ export default class ConsentManagerBuilder extends Component<Props, State> {
       preferences = destinationPreferences || initialPreferences
     }
 
-    conditionallyLoadAnalytics({
-      writeKey,
-      destinations,
-      destinationPreferences,
-      isConsentRequired,
-      defaultDestinationBehavior,
-      categoryPreferences: preferences
-    })
+    if (!initialPreferences) {
```

Comment:
@felipe-najson-ntf This line could be problematic. We want to load `analytics.js` even if the user doesn't provide the `initialPreferences`. I think adding this `if` clause will lead to cases where users already consented to everything, and their consent settings is stored in cookies, but then the `analytics.js` doesn't load.

Reply:
This is the work in progress branch: 
<img width="1914" alt="Storybook" src="https://user-images.githubusercontent.com/2992515/140589183-dfeed232-e8cc-4cb0-9efd-14fb556c75f3.png">

Reply:
This is the master branch: 
<img width="1909" alt="Storybook" src="https://user-images.githubusercontent.com/2992515/140589271-0041c81d-d98b-4902-bac7-09a4d2331363.png">

Reply:
Good morning @pooyaj, I do not agree with you on this point. I think that is exactly what users are complaining about, analytics.js initializes automatically without user consent. The idea is to load the initial preferences to the cookie "tracking-preferences" but NOT initialize. 

With this we tried not to affect other behaviors of the library except when there are Initial preferences, do not execute it. With this solution when loading the page we will have:
![image](https://user-images.githubusercontent.com/89416739/140937479-5a422843-ca11-400a-9c2c-8547b0422594.png)

Once the consent is accepted in the modal, tracking starts:
![image](https://user-images.githubusercontent.com/89416739/140937629-0a7ec481-c993-4795-9e7e-f4e3d3246607.png)

**_Note: I have changed the if sentence to something more precise in the following commit._**

Reply:
@pooyaj I just left a pr with the new solution. If you still have doubts of why I included the if sentence we can have a small meeting to conclude with this issue. Greetings

Reply:
Sorry for another long delay for the response. I think there is a misunderstanding about the purpose of the `initialPreferences`. When we set the `initialPreferences`, it means that we are **automatically opt-ing in** users to those preferences only if there is no prior consent from the user. It means we assume consent from the user according to the `initialPreferences`. 

Another important point here is that if there is already a consent ( in the cookies, or from the initial preferences ) we want to attempt to call `conditionallyLoadAnalytics`. If we don't do that, user ends up in a situation where they have to consent everytime they refresh the page ( or every time the consent manager component mounts )